docs/specs/sequences/writeback.md

```diff
@@ -30,12 +51,21 @@ While the system should never abort trying to reconcile the config, after 1 minu
 
 In the case of any of the error states (failure, invalid, overridden), the [status](../../messages/status.md) field for the point should be populated to provide additional debugging information about the error.
 
-## State Etag Example
+## State Etag
```

Comment:
should this be ###?  Seems like it's a subsection of device behavior?  THen you have
state --> value_state
state --> state_etag